preprocessy/encoding/_categorical.py

```diff
@@ -38,6 +38,10 @@ def __init__(self, params, target_label=None, train_df=None, test_df=None):
         if "ord_dict" in params.keys():
             self.ord_dict = params["ord_dict"]
 
+    def __repr__(self):
+        params_default = dict(([], {}))
+        return f"EncodeData(params={params_default},train_df={self.train_df},test_df={self.test_df})"
```

Comment:
Why is ```self.params``` not being used here for the params argument?
Please remove the train and test dataframe from here as well, I have mentioned the reason in another comment.

Reply:
actually, params is of the form, 
params={
cat_cols = []
ord_dict = {}
}

So I made a default empty dictionary for that.